roxctl/common/auth/token_test.go

```diff
@@ -28,6 +31,22 @@ func (s *tokenSuite) Test_RetrieveAuthToken_WithEnv() {
 	s.Equal(got, testTokenVal, "Did not receive correct rox auth token from environment")
 }
 
+func (s *tokenSuite) Test_RetrieveAuthToken_WithFileEnv() {
+	tstDir, err := ioutils.TempDir("", "roxctl-test-common-auth")
```

Comment:
Let's not create a new dependency for `containers/storage` and just use `s.T().TempDir()` instead.